.github/workflows/ci.yml

```diff
@@ -60,28 +63,31 @@ jobs:
         with:
           node-version: ${{ env.NODE_VERSION }}
 
-      - name: Get Yarn cache path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: Update npm
+        run: npm i -g npm@latest
+
+      - name: Get npm cache path
+        id: npm-cache-dir-path
+        run: echo "::set-output name=dir::$(npm config get cache)"
 
-      - name: Cache Yarn cache and node_modules
+      - name: Cache npm cache and node_modules
         id: cache-dependencies
         uses: actions/cache@v2
         with:
           path: |
-            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+            ${{ steps.npm-cache-dir-path.outputs.dir }}
```

Comment:
Similarly to above, can you check if we can write just `~/.npm` and remove lines 66-72?